src/main/scala/com/kinja/config/LiftedTypesafeConfig.scala

```diff
@@ -70,6 +70,7 @@ final case class LiftedTypesafeConfig private[LiftedTypesafeConfig] (configAndNa
   def getObject(name : String) : BootupErrors[ConfigObject] = lift(name, underlying.getObject)
 
   /** Retrieves the value found at the given name as a ConfigObject. */
+  @SuppressWarnings(Array("org.wartremover.warts.Nothing"))
```

Comment:
Explicit type annotation doesn't help because the signature of `getObjectList` itself contains `Nothing`.